packages/modules/b2c-core/src/api/vendor/product-types/validators.ts

```diff
@@ -1,17 +1,44 @@
-import { z } from 'zod'
+import { z } from "zod";
 
-import { createFindParams } from '@medusajs/medusa/api/utils/validators'
+import { createFindParams } from "@medusajs/medusa/api/utils/validators";
+
+const dateFilterSchema = z
```

Comment:
Maybe here you can use Medusa's createOperatorMap() function from their API utils?